### PR TITLE
Add terraform blogpost

### LINKED ISSUE #342: A Multi-Environment Terraform Setup
https://hackmd.io/zoofjoYZSLa1RGuT3W0HaQ

Need to have this reviewed before opening a PR

### PR DESCRIPTION
**Pull request by jorgegonzalez**
Closes #342

### CODE REVIEW

File: _posts/2021-06-12-multi-environment-terraform.md [hunk 1]
```diff
@@ -0,0 +1,274 @@
+---
+layout: post
+title: "A Multi Environment Terraform Setup"
+short_title: "Multi Environment Terraform"
+date: 2021-06-12
```
Comment (jtwillis92) [inline _posts/2021-06-12-multi-environment-terraform.md:5]: is this YYYY-MM-DD or YYYY-DD-MM?

File: _posts/2021-06-12-multi-environment-terraform.md [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+author: Jorge Gonzalez
+---
+
+# A Multi-Environment Terraform Setup
+
+
+## Terraform 
+We adopted [Terraform](https://www.terraform.io/) recently to help in the provisioning of our service infrastructure across multiple environments. We developed this workflow early in our application’s lifecycle, an advantage that will yield the ability to create and test new services in lower, non-production environments, and then easily replicate them to higher environments.
+
+It is well described elsewhere how to manually manage such deployments across environments, but this deployment workflow is notably entirely automated, managed only by  commiting the declarative, high-level configuration [Terraform language](https://www.terraform.io/docs/language/index.html) syntax with which we describe our infrastructure; deployments are managed by certain git merges, and services can be updated and created with changes to our Terraform configuration files.
```
Comment (jtwillis92) [inline _posts/2021-06-12-multi-environment-terraform.md:18]:
```suggestion
It is well described elsewhere how to manually manage such deployments across environments, but this deployment workflow is notably entirely automated, managed only by committing the declarative, high-level configuration [Terraform language](https://www.terraform.io/docs/language/index.html) syntax with which we describe our infrastructure; deployments are managed by certain git merges, and services can be updated and created with changes to our Terraform configuration files.
```

File: _posts/2021-06-12-multi-environment-terraform.md [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+```yaml
+# .circle/config.yml
+
+# ...
+
+workflows:
+  dev-deployment:
+    jobs:
+      - deploy-infrastructure:
+			filters:
```
Comment (jtwillis92) [inline _posts/2021-06-12-multi-environment-terraform.md:153]: nit: indentation

File: _posts/2021-06-12-multi-environment-terraform.md [hunk 4]
```diff
@@ -0,0 +1,274 @@
...
```
Comment (jtwillis92) [inline _posts/2021-06-12-multi-environment-terraform.md:218]: nit: indentation

**Review by jtwillis92 — APPROVED** [review]
This is excellent, very well written and concise!

**Review by abottoms-coder — APPROVED** [review]

**Comment by abottoms-coder** [comment]
Looks great, @jorgegonzalez !

File: _posts/2021-06-12-multi-environment-terraform.md (hunk 1 above)
Comment (BarakStout) [reply _posts/2021-06-12-multi-environment-terraform.md:5]: @jorgegonzalez format should be YYYY-MM-DD

File: _posts/2021-06-12-multi-environment-terraform.md (hunk 3 above)
Comment (BarakStout) [reply _posts/2021-06-12-multi-environment-terraform.md:153]: @jorgegonzalez ?

File: _posts/2021-12-06-multi-environment-terraform.md [hunk 5]
```diff
@@ -0,0 +1,274 @@
+---
+layout: post
+title: "A Multi Environment Terraform Setup"
+short_title: "Multi Environment Terraform"
+date: 2021-12-06
+categories:
+featimg:
+bgimg:
```
Comment (BarakStout) [inline _posts/2021-12-06-multi-environment-terraform.md:8]: Needs image

File: _posts/2021-12-06-multi-environment-terraform.md [hunk 6]
```diff
@@ -0,0 +1,274 @@
+---
+layout: post
+title: "A Multi Environment Terraform Setup"
+short_title: "Multi Environment Terraform"
+date: 2021-12-06
+categories:
+featimg:
```
Comment (BarakStout) [inline _posts/2021-12-06-multi-environment-terraform.md:7]: Needs image

**Review by BarakStout — APPROVED** [review]
LGTM 👍

**Pull request merged**